src/components/activities.js

```diff
@@ -314,7 +317,22 @@ export class ActivityPorts {
     if (addDefaultArguments) {
       args = this.addDefaultArguments(args);
     }
-    return this.openActivityIframePort_(iframe, url, args);
+    return this.deps_
+      .storage()
+      .get(Constants.USER_TOKEN, true)
```

Comment:
nit: Add inline comment explaining mean of true literal.

Reply:
Done.